paddle/fluid/framework/details/parallel_ssa_graph_executor.cc

```diff
@@ -13,11 +13,70 @@
 // limitations under the License.
 
 #include "paddle/fluid/framework/details/parallel_ssa_graph_executor.h"
+#include "paddle/fluid/framework/ir/graph_helper.h"
 
 namespace paddle {
 namespace framework {
 namespace details {
 
+std::vector<std::unique_ptr<ir::Graph>> SeparateMultiDevicesGraph(
```

Comment:
This should be in a pass?

Reply:
let's hide this in parallel graph executor for now. Splitting graph is not safely supported now. (add comment). We might want to remove this.